internal/rand/random.go

```diff
@@ -302,6 +314,8 @@ func (r *Rand) Perm(n int) []int {
 // NOTE: This relies on the os's random number generator.
 // For real security, we should salt that with some seed.
 // See github.com/cometbft/cometbft/crypto for a more secure reader.
```

Comment:
I disagree with this comment, but will leave to the general cleanup PR for this package to remove.

Reply:
This comment does not make sense given the preamble of this source file...